public/keymaps/heliar_wm1_hotswap_default.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard": "heliar/wm1_hotswap",
+  "keymap": "default",
```

You can't actually use `default` as the value for `keymap`, because Configurator's API can see the `keymap.c` files that are in qmk_firmware, even though Configurator itself can't do anything with them. In fact, the value for `keymap` cannot match *any* `keymap.c` that exists in qmk_firmware, for the same reason. The API tries to create a `keyboards/{keyboard}/keymaps/{keymap}/keymap.c` file when the compile job completes, but if one already exists the operation fails.

I append a partial commit hash to the `keymap` value to prevent this.

```suggestion
  "keymap": "default_2b318ba",
```